sdk_extensions/zpages/src/main/java/io/opentelemetry/sdk/extensions/zpages/TracezDataAggregator.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.extensions.zpages;
+
+import io.opentelemetry.sdk.trace.ReadableSpan;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.trace.Status.CanonicalCode;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * A data aggregator for the traceZ zPage.
+ *
+ * <p>The traceZ data aggregator compiles information about the running spans, span latencies, and
+ * error spans for the frontend of the zPage.
+ */
+@ThreadSafe
+final class TracezDataAggregator {
+  private final TracezSpanProcessor spanProcessor;
+
+  /**
+   * Constructor for {@link io.opentelemetry.sdk.extensions.zpages.TracezDataAggregator}.
```

Comment:
There's a bunch of fully-qualified names in docs throughout this PR. Let's make sure to use the imports and remove the FQNs when we can.

Reply:
Removed the FQNs from the span processor and data aggregator